schema.go

```diff
@@ -106,3 +113,28 @@ func findBuildByBuilder(db *mgo.Database, builder string, buildnum int) (*LogKee
 	}
 	return build, nil
 }
+
+func GetOldTests(db *mgo.Database, now time.Time) (*[]Test, error) {
+	query := bson.M{
+		"started": bson.M{"$lte": now.Add(-deletePassedTestCutoff)},
+	}
+	tests := []Test{}
+	err := db.C("tests").Find(query).Sort("-started").Limit(maxTests).All(&tests)
+	if err != nil {
+		return nil, errors.Wrap(err, "error finding tests")
+	}
+	return &tests, err
+}
+
+func CleanupOldLogsByTest(db *mgo.Database, id bson.ObjectId) error {
+	err := db.C("tests").RemoveId(id)
+	if err != nil {
+		return errors.Wrap(err, "error deleting test")
+	}
+
+	_, err = db.C("logs").RemoveAll(bson.M{"test_id": id})
```

Comment:
should the collection names here be constants? key names? 

does it make sense to pass the ID as a string? what is the type in the struct? 

are we sure that this is logged/

Reply:
In the Log struct, test_id is: `	TestId  *bson.ObjectId `bson:"test_id"`
So i think it makes more sense not to pass this id as a string.
What do you mean are we sure that this is logged? As in is the testID in logs populated? That would be our only way to connect the test to the log.